src/js/button.js

```diff
@@ -1,3 +1,8 @@
+import Component from './component';
+import * as Lib from './lib';
```

Comment:
It could just as well just be named `browser-checks.js`. Having it be under `util` is serving no purpose.